packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.js

```diff
@@ -483,6 +515,10 @@ export const StaticProperties: ObjectMap<
 };
 
 export const InstanceProperties = {
+  asIndexedPairs: define("instance/asIndexedPairs", [
+    "esnext.async-iterator.as-indexed-pairs",
+    "esnext.iterator.as-indexed-pairs",
```

Comment:
The most `core-js` modules have dependencies. Check related `core-js` entry points like this https://github.com/zloirock/core-js/blob/master/packages/core-js/features/async-iterator/as-indexed-pairs.js.